tests/handlers/test_oidc.py

```diff
@@ -832,7 +832,7 @@ def test_map_userinfo_to_user_retries(self):
         # test_user is already taken, so test_user1 gets registered instead.
         self.assertEqual(mxid, "@test_user1:test")
 
-        # Register all of the potential users for a particular username.
+        # Register all of the potential usernames for a particular username.
```

Comment:
```suggestion
        # Register all of the potential mxids for a particular OIDC username.
```

(similarly for SAML)